recipes/lief/all/patches/001_link_to_conan.patch

```diff
@@ -83,38 +85,42 @@
  if (LIEF_LOGGING)
    source_group("easylogging"     FILES ${ELG_SOURCE_DIR}/easylogging++.cc)
  endif()
-@@ -316,7 +303,7 @@
+@@ -316,7 +303,8 @@ target_include_directories(LIB_LIEF
    PRIVATE "${LIEF_PRIVATE_INCLUDE_DIR}")
-
+ 
  if (LIEF_ENABLE_JSON)
 -  add_dependencies(LIB_LIEF lief_libjson)
-+  target_link_libraries(LIB_LIEF CONAN_PKG::nlohmann_json)
```

Comment:
Is it really required? CONAN_PKG will be deprecated soon.

Reply:
I agree with you.
This line is deleted by this PR.